app-typescript/components/ToggleBox/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {SimpleToggleBox} from './SimpleToggleBox';
+import {CustomHeaderToggleBox} from './CustomHeaderToggleBox';
+
+export interface IPropsWithLabel {
+    variant: 'simple';
+    title: string;
+    badge?: JSX.Element;
+    children: React.ReactNode;
+    hideUsingCSS?: boolean;
+    initiallyOpen?: boolean; // defaults to false
+    className?: string;
+    margin?: 'none' | 'small' | 'normal' | 'large';
+    onOpen?(): void;
+    onClose?(): void;
+}
+
+export interface IPropsWithoutLabel {
+    variant: 'custom-header';
+    // always visible
```

Comment:
![image](https://github.com/superdesk/superdesk-ui-framework/assets/2076953/11265739-9e31-401f-b5e6-54f5029ef5c3)

use line breaks to make it more clear which line a comment applies. In this case, since the comment is short it's even better to put it on the same line.